web/static/js/components/simulation/VoiceWindow.jsx

```diff
@@ -74,25 +80,48 @@ const VoiceWindow = translate()(class extends Component<VoiceWindowProps> {
         this.playIVR(ivr)
       }
     } else {
-      this.spectrum.stop()
+      if (this.props.readOnly) return // simulation is not active so no need for repeating the audio
+
+      // no more prompts to play so we start countdown to repeat the last one
+      if (this.audio) {
+        if (this.repeatMessageTimer) return // if already repeating then return
+
+        // we break the audio play chain because is the last message and we start repeating it
+        this.audio.onended = null
+
+        this.repeatMessageTimer = setInterval(() => {
+          if (this.timesRepeated >= 3) {
+            this.hangUp()
+            return
+          }
+
+          this.audio.play()
+          this.timesRepeated += 1
+        }, 5000)
```

Comment:
Maybe extract a `repeat()` method? So we could:

```js
if (ivr) {
  // ...
} else {
  this.spectrum.stop()

  if (!this.props.readOnly) {
    this.repeat()
  }
}
```

Reply:
There can be an issue with `setInterval()` because a question may be very long (see a questionnaire in staging), so we should just start a timeout after the question has finished, or aborting if it's the third time.